after.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# The required environment variables are:
+# user - e.g. 'pjmhill'
+
+# optional variables for github commit status API
+# github_token - the token that is used to push commit status
+
+# require to check status
+# result - the result of the jenkins_build.sh -- needs to be captured in Jenkins' cell
+
+
+stack=dev
```

Comment:
agree. I didn't see that you are reviewing. I'm also moving before.sh and after.sh to jenkins_build.sh. What do you think about moving stack to jenkins_build.sh since it's also defined in docker_build.sh